src/javafundamentals/interfaces/Ex01/SalesMethods.java

```diff
@@ -0,0 +1,8 @@
+package javafundamentals.interfaces.Ex01;
+
+public interface SalesMethods {
```

Comment:
Try to think about naming as close as possible to the real world.
This interface contains methods that are for selling items.
The items that are for sale will implement this interface.
A better name would be something like:
`SellableItem`

Reply:
okay, I'll really work on my naming- thanks!